Mathlib/Analysis/Asymptotics/Lemmas.lean

```diff
@@ -227,7 +227,7 @@ theorem IsBigO.const_smul_left (h : f' =O[l] g) (c : R) : (c • f') =O[l] g :=
 theorem IsLittleO.const_smul_left (h : f' =o[l] g) (c : R) : (c • f') =o[l] g :=
   (IsBigO.const_smul_self _).trans_isLittleO h
 
-variable [Module 𝕜 E'] [IsBoundedSMul 𝕜 E']
+variable [Module 𝕜 E'] [NormSMulClass 𝕜 E']
```

Comment:
Could you modify the docstring of `IsBoundedSMul` to remark that when working with a module over a division ring, it is preferable to use the mathematically equivalent `NormSMulClass` instead?